packages/go/slices/slices.go

```diff
@@ -82,6 +82,7 @@ func UniqueBy[T any, U comparable](slice []T, fn func(T) U) []T {
 	return out
 }
 
+// TODO delete this function, use utils.Contains instead
```

Comment:
I think it would be best to just remove this `TODO` for now and make a case for the change you propose outside of this PR.